README.md

```diff
@@ -1 +1,19 @@
-# sig-ux
\ No newline at end of file
+# SIG UX
```

Comment:
https://www.jenkins.io/sigs/ux/ should get updated too. I think you should actually link there